config/development.yaml

```diff
@@ -14,6 +14,7 @@ workflowTerminateDisabled: false
 workflowCancelDisabled: false
 workflowSignalDisabled: false
 workflowResetDisabled: false
+bulkActionsDisabled: false
```

Comment:
@flossypurse @DJSanti do we communicate this to users as batch operations or bulk actions? In API we call this batch jobs / operations

Reply:
@feedmeapples, @flossypurse and @DJSanti - some context. Internally we are differentiating bulk and batch as 2 different things. 

Bulk - triggering an action on a known set of `n` workflow executions where `n` could be anywhere from `1` to the `500` (depending on the page size the user has set)
Batch - triggering an action against _all_ workflow executions matching a query

The intent here is to mimic Gmail when your search matches more results than you can view on a "page"
<img width="1701" alt="Screen Shot 2023-01-31 at 3 57 04 PM" src="https://user-images.githubusercontent.com/11775628/215903512-6783c391-84bc-44a8-ba28-aa23780ca50f.png">

The purpose of this flag is to disable both, so I'm happy to rename it to `batchActionsDisabled` to align with other tools. Personally I'm against differentiating batch and bulk externally to users. I think of them as the same feature even though the internal implementation might be slightly different.

Reply:
thx for the context, bulk sounds good to me

Reply:
@feedmeapples I actually prefer batch now that you've mentioned it, but just wanted to clarify that although we differentiate internally I think it's ok for this 1 config value to control both batch and bulk features in the UI 👍